master_changes.md

```diff
@@ -55,6 +55,7 @@ users)
 ## Exec
 
 ## Source
+  * Avoid fetching with `--depth 1` for `opam source` command [#5888 @moyodiallo - fix #5061]
```

Comment:
It also needs changes in the API section, for ex
```
  * `OpamRepositoryBackend.pull_url`, `OpamCS.fetch`: add `full_fetch` optional argument to pull full history if url is a VCS`
```
And for reftest, in Reftests section

Reply:
It is also better to have the changes log in the same commit where the changes are done.

Reply:
Done here https://github.com/ocaml/opam/pull/5888/commits/54b827dfeed33e589757e965b862dda130367d4f.